packages/manager/apps/dedicated/client/app/ip/components/list/list.html

```diff
@@ -336,11 +336,18 @@
                     </small>
                 </td>
 
-                <!-- Campus -->
-                <td
-                    class="text-center"
-                    data-ng-bind="ipBlock.campus ? ipBlock.campus : '-'"
-                ></td>
+                <!-- Region -->
+                <td class="text-center">
+                    <span
+                        class="d-block"
+                        data-ng-repeat="region in ipBlock.regions track by $index"
+                        data-translate="{{ :: 'ip_region_' + region }}"
+                    ></span>
+                    <span
+                        data-ng-if="!ipBlock.regions"
+                        data-ng-bind="'-'"
+                    ></span>
+                </td>
```

Comment:
Maybe you should create a small component to avoid duplicated HTML code, what do you think ?

Reply:
I'm not sure I'll gain much because the absence of duplication will be compensated for by more files.
The other <td> are also duplicated for ip and ipblock.